tests/Functional/Driver/PDO/PgSQL/DBAL6024Test.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Doctrine\DBAL\Tests\Functional\Driver\PDO\PgSQL;
+
+use Doctrine\DBAL\Schema\Comparator;
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+
+/**
+ * How to run this test with PostgreSQL (the original target of the issue):
+ * 1) Start a PostgreSQLInstance
+ * 2) If needed, change ci/github/phpunit/pdo_pgsql.xml according to your PostgreSQL local settings
+ * 3) Run
+ * vendor/bin/phpunit -c ci/github/phpunit/pdo_pgsql.xml tests/Functional/Driver/PDO/PgSQL/DBAL6024Test.php
+ */
```

Comment:
This applies to all tests. Generic instructions on how to run the test suite could be added to the documentation, but they don't belong here.

Reply:
I agree. Where is the best place to add this kind of instruction?

Reply:
Somewhere in the documentation, I suppose. There's a "docs" folder in this repository. But that should be a separate PR, please.